src/test_yang.act

```diff
@@ -230,6 +231,26 @@ def _test_parse_pryang_submodule():
     yang_out = y.pryang()
     testing.assertEqual(original, yang_out, "YANG round trip failed\n" + original + "\n===\n" + yang_out)
 
+def _test_snode_to_statement():
+    original = test_yang
+    y = yang.parser.parse(original)
+    n = yang.schema.stmt_to_snode(y)
+    yy = n.to_statement()
+    return yy.pryang()
+
+def _test_parse_snode_pryang_twice():
+    # YANG -> Statement -> SchemaNode -> Statement -> YANG -> Statement -> SchemaNode -> Statement -> YANG
+    original = test_yang
+    y1 = yang.parser.parse(original)
+    n1 = yang.schema.stmt_to_snode(y1)
```

Comment:
We should totally add a convenience function in the yang.act function for this!!
```python
def parse(yang_text: str) -> yang.schema.SchemaNode:
    ys = yang.parser.parse(yang_text)
    return yang.schema.stmt_to_snode(ys)
```